docs/source/explore/get-started/install.rst

```diff
@@ -53,70 +75,80 @@ To install on Windows, simply double-click on the downloaded installer file to o
 
+Vircadia is always changing, as we work to improve performance and add features that will enhance your experience in the metaverse. At any time, you can download the latest release from our `website <https://vircadia.com/download-vircadia/#interface>`_.
 
-.. Vircadia is always changing, as we work to improve performance and add features that will enhance your experience in the metaverse. If a new version has been released, you will be prompted to upgrade your installation the next time you run Vircadia. At any time, you can also download the latest release from our `website <https://www.highfidelity.com/download>`_.
-
-.. You cannot upgrade if Sandbox or the Console is running in the background of your computer. Be sure to quit these applications before upgrading.
-
-.. For Windows, locate the Vircadia app in your system tray. Right-click on the icon and select 'Quit'. Alternatively, end the 'server-console' background process using the Task Manager.
-.. For OS X, locate the Vircadia icon in the menu bar. Click on the icon and select 'Quit'.
+You cannot upgrade Vircadia on Windows if you have a server running in the background of your computer. Be sure to quit this application before upgrading.
 
-.. For more information on the latest releases, see our :doc:`Release Notes <../../release-notes>`.
+For Windows, locate the Vircadia app in your system tray. Right-click on the icon and select 'Quit'. Alternatively, end the 'server-console' background process using the Task Manager.
 
+For more information on the latest releases, see our :doc:`Release Notes <../../release-notes>`.
 
 
 ---------------------------------
 Perform a Clean Install
 ---------------------------------
 
-If you're facing problems when you load Interface and Sandbox, you can try performing a clean install. A clean install removes multiple files and settings that you may need once you install Vircadia again. 
+If you're facing problems when you load Interface or the local Server, then you can try performing a clean install. A clean install removes multiple files and settings that you may need once you install Vircadia again.
 
 .. note:: Ensure that you back up the following files before a clean install: Favorites, Wearables, Sandbox, and Entities. These settings will be deleted during the clean install.
 
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 Windows Clean Install
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
-1. Click on the Start menu and type "Add or Remove Programs" in your Windows search bar.  
-2. Uninstall any versions of Vircadia that are visible (Including any Steam installs). 
-3. Once Vircadia is uninstalled, browse to your %Program Files% directory. Delete all folders related to Vircadia. If you installed through Steam, these folders will be located at ``C:/Program Files(x86)/Steam/steamapps/common``. 
+1. Click on the Start menu and type "Add or Remove Programs" in your Windows search bar.
+2. Uninstall any versions of Vircadia that are visible (Including any Steam installs).
```

Comment:
```suggestion
2. Uninstall any versions of Vircadia that are visible (including any Steam installs).
```

Reply:
I removed this section now, as it suggests Vircadia being available on Steam, which it isn't and probably won't be in the near future.